Extensions/Physics3DBehavior/JsExtension.js

```diff
@@ -0,0 +1,2463 @@
+          if (newValueAsNumber !== newValueAsNumber) return false;
+          behaviorContent
+            .getChild('shapeDimensionB')
+            .setDoubleValue(newValueAsNumber);
+          return true;
+        }
+
+        if (propertyName === 'shapeDimensionC') {
+          const newValueAsNumber = parseFloat(newValue);
+          if (newValueAsNumber !== newValueAsNumber) return false;
+          behaviorContent
+            .getChild('shapeDimensionC')
+            .setDoubleValue(newValueAsNumber);
+          return true;
+        }
+
+        if (propertyName === 'density') {
+          behaviorContent
+            .getChild('density')
+            .setDoubleValue(parseFloat(newValue));
+          return true;
+        }
+
+        if (propertyName === 'friction') {
+          const newValueAsNumber = parseFloat(newValue);
+          if (newValueAsNumber !== newValueAsNumber) return false;
+          behaviorContent.getChild('friction').setDoubleValue(newValueAsNumber);
+          return true;
+        }
+
+        if (propertyName === 'restitution') {
+          const newValueAsNumber = parseFloat(newValue);
+          if (newValueAsNumber !== newValueAsNumber) return false;
+          behaviorContent
+            .getChild('restitution')
+            .setDoubleValue(newValueAsNumber);
+          return true;
+        }
+
+        if (propertyName === 'linearDamping') {
+          const newValueAsNumber = Math.max(0, parseFloat(newValue));
+          if (newValueAsNumber !== newValueAsNumber) return false;
+          behaviorContent
+            .getChild('linearDamping')
+            .setDoubleValue(newValueAsNumber);
+          return true;
+        }
+
+        if (propertyName === 'angularDamping') {
+          const newValueAsNumber = Math.max(0, parseFloat(newValue));
+          if (newValueAsNumber !== newValueAsNumber) return false;
+          behaviorContent
+            .getChild('angularDamping')
+            .setDoubleValue(newValueAsNumber);
+          return true;
+        }
+
+        if (propertyName === 'gravityScale') {
+          const newValueAsNumber = parseFloat(newValue);
+          if (newValueAsNumber !== newValueAsNumber) return false;
+          behaviorContent
+            .getChild('gravityScale')
+            .setDoubleValue(newValueAsNumber);
+          return true;
+        }
+
+        if (propertyName === 'layers') {
+          behaviorContent
+            .getChild('layers')
+            .setIntValue(parseInt(newValue, 10));
+          return true;
+        }
+
+        if (propertyName === 'masks') {
+          behaviorContent.getChild('masks').setIntValue(parseInt(newValue, 10));
+          return true;
+        }
+
+        return false;
+      };
+      behavior.getProperties = function (behaviorContent) {
+        const behaviorProperties = new gd.MapStringPropertyDescriptor();
+
+        behaviorProperties
+          .getOrCreate('Object3D')
```

Comment:
Should be:
```suggestion
          .getOrCreate('object3D')
```

? It's not consistent  with the updateProperty